smac/tae/execute_ta_run.py

```diff
@@ -198,6 +214,7 @@ def start(self, config: Configuration,
                                     "in the trajectory-file.")
 
         if self.run_obj == "runtime":
+            assert cutoff is not None  # please mypy
```

Comment:
Could you catch this with if and raise an exception

Reply:
I added a raise further up in the code prior to calling `super().run()`